_trans/th.json

```diff
@@ -0,0 +1,42 @@
+	"description": "รายการdirect linkเพื่อลบบัญชีบนเว็ปไซต์และสื่อสังคมออนไลน์ ค้นพบวิธีลบบัญชีFacebook, Twitter, LinkedInและบริการอื่นๆได้ที่นี่",
+	"difficulty": "ความยาก",
+	"difficulty_easy": "ง่าย",
+	"difficulty_medium": "ปานกลาง",
+	"difficulty_hard": "ยาก",
+	"difficulty_impossible": "เป็นไปไม่ได้",
+	"tagline": "รายการdirect linkสำหรับลบบัญชีจากโลกออนไลน์์",
+	"noinfo": "ไม่มีข้อมูล",
+	"showinfo": "แสดงข้อมูล...",
+	"hideinfo": "ซ่อนข้อมูล...",
+	"whatisthis": "นี่คืออะไร?",
+	"whatisthis1": "หลายบริษัทใช้เทคนิค <a href=\"http://darkpatterns.org/\">dark pattern</a> เพื่อไม่ให้ผู้ใช้สามารถหาวิธีลบบัญชีออกได้โดยง่าย JustDelete.me ต้องการจะสร้างรายการurlเพื่อให้คุณสามารถลบบัญชีเหล่านี้ได้อย่างง่ายดาย",
+	"whatisthis2": "มีวิธีการลบบัญชีที่อยากจะเพิ่มเติม? <a href=\"http://github.com/rmlewisuk/justdelete.me\">ForkโครงการจากGitHubสิ</a>.",
+	"whatisthis3": "<em>ยกเลิกการรายงานผ่านอีเมล์ช่าวคราว</em>",
+	"guide": "คำแนะนำ",
+	"guideexplanations": "ลิ้งข้างบนแสดงความยากง่ายในการลบบัญชีด้วยสีที่แตกต่างกัน:",
+	"guideeasy": "<span class=\"green\">ง่าย</span> - ขั้นตอนง่ายๆ",
+	"guidemedium": "<span class=\"yellow\">ปานกลาง</span> - ต้องออกแรงนิดหน่อย",
+	"guidehard": "<span class=\"red\">ยาก</span> - ไม่สามารถลบได้โดยไม่ติดต่อฝ่ายบริการลูกค้า",
+	"guideimpossible": "<span class=\"black\">เป็นไปไม่ได้</span> - ไม่สามารถลบได้",
+	"translationcontrib": "ผู้มีส่วนร่วมในการแปล",
+	"morecontrib": "ผู้มีส่วนร่วมท่านอื่นๆ",
+	"viewcontrib": "ผู้มีส่วนร่วมทั้งหมด",
+	"extension": "ส่วนเสริมGoogle Chrome",
+	"extensionguide": "คำแนะนำ'จุด'ของส่วนเสริม",
+	"extensionp1": "คุณ <a target=\"_blank\" href=\"http://mikerogers.io\">Mike Rogers</a> ผู้ใจดีได้ช่วยเหลือเราสร้างส่วนเสริมGoogle Chromeอันยอดเยี่ยมสำหรับJustDelete.meขึ้น",
+	"extensionp2": "เมื่อคุณกำลังใช้งานเว็ปไซต์ที่justdelete.meมีข้อมูลอยู่ ส่วนเสริมGoogle Chromeจะแสดง'จุด'ที่omnibar คลิ๊กที่'จุด'ดังกล่าวจะนำคุณไปสู่หน้าสำหรับการลบบัญชี",
+	"extensionp3": "ต้องการติดตั้งส่วนเสริม? กรุณาไปที่นี่ <a target=\"_blank\" href=\"https://chrome.google.com/webstore/detail/justdeleteme/hfpofkfbabpbbmchmiekfnlcgaedbgcf\">Chrome Web Store</a>.",
+	"extensionl1": "<span class=\"dot-wrapper\"><span class=\"dot easy\"></span></span> - ขั้นตอนง่ายๆ",
+	"extensionl2": "<span class=\"dot-wrapper\"><span class=\"dot medium\"></span></span> - ต้องออกแรงนิดหน่อย",
+	"extensionl3": "<span class=\"dot-wrapper\"><span class=\"dot hard\"></span></span> - ไม่สามารถลบได้โดยไม่ติดต่อฝ่ายบริการลูกค้า",
+	"extensionl4": "<span class=\"dot-wrapper\"><span class=\"dot impossible\"></span></span> - ไม่สามารถลบได้",
+	"banner": "ลอง<span>ส่วนเสริมGoogle Chrome</span>ตัวใหม่ของเราสิ",
+	"footer": "สร้างสรรค์โดย <a href=\"http://robblewis.me\">Robb Lewis</a> และ <a href=\"http://edpoole.me\">Ed Poole</a> | Forkบน<a href=\"http://github.com/rmlewisuk/justdelete.me\">GitHub</a> | hostบน<a href=\"https://www.digitalocean.com/?refcode=8e1d8283bd20b\">DigitalOcean</a> | วิเคราะห์ข้อมูลด้วย<a href=\"https://t.co/fvPnva7p4Y\">GoSquared</a>",
```

Comment:
Please  remove everything after "| hostบ", including it, as this is not relevant anymore.

Change the fork link to https://github.com/jdm-contrib/justdelete.me